packages/mobile/src/geth/geth.ts

```diff
@@ -279,15 +282,13 @@ async function writeStaticNodes(nodeDir: string, enodes: string) {
   await RNFS.writeFile(staticNodesFile, enodes, 'utf8')
 }
 
-async function attemptGethCorruptionFix(geth: any, sync: boolean = true) {
+async function attemptGethCorruptionFix(geth: any, shouldStartNodec: boolean = true) {
   const deleteChainDataResult = await deleteChainData()
   const deleteGethLockResult = await deleteGethLockFile()
   if (deleteChainDataResult && deleteGethLockResult) {
     await geth.start()
     gethInstance = geth
-    if (sync) {
```

Comment:
why remove this?

Reply:
This method only gets called when an error happens when trying to start geth. Before we were starting geth both in normal mode and data-saver mode. But with the new changes we don't start geth at all when in `data-saver` mode so this method shouldn't ever be called in `data-saver` mode therefor it doesn't need to split behaviour anymore based on the mode. If we ever reach this code we've had to start geth therefore we are syncing.